src/styles/utils/__fonts.scss

```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: "Roboto";
+  src: url(../fonts/Roboto/Roboto-Bold.ttf);
+  font-weight: 700;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url(../fonts/Roboto/Roboto-Medium.ttf);
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url(../fonts/Roboto/Roboto-Regular.ttf);
+  font-weight: 400;
+}
```

Comment:
You could just add these fonts in HTML instead of creating a new file and typing excessive code, but it's not a big problem.

Reply:
i know, but i like when fonts are as separate file